catalog/transformer/catalog-transformer-csv-common/pom.xml

```diff
@@ -44,5 +44,52 @@
             <groupId>ddf.platform.util</groupId>
             <artifactId>platform-util</artifactId>
         </dependency>
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+            <version>2.2.11</version>
```

Comment:
Does this need to be a different version then the version declared in the root pom?

Reply:
This matches the version that is pulled in indirectly via the platform dependencies.  The jaxb version defined in the root pom is not for this dependency, even though you would think that it would be. 🤷